# HG changeset patch
# Parent  903589eabf9b28f8e6701f75a80131042bdd118d
Don't log device labels as tags when routing packets

diff --git a/core/router.c b/core/router.c
--- a/core/router.c
+++ b/core/router.c
@@ -174,7 +174,7 @@
     }
 
     if (pkt_infos) {
-        lgtd_info("sending %s to #%s", pkt_infos->name, label);
+        lgtd_info("sending %s to %s", pkt_infos->name, label);
     }
 }
 
